rollup.config.js

```diff
@@ -32,7 +34,13 @@ function createESMConfig(input, output) {
     external,
     plugins: [
       resolve({ extensions }),
-      typescript(),
+      esbuild({
+        minify: process.env.NODE_ENV === 'production',
```

Comment:
I'm not sure where this NODE_ENV is coming from. Is this `true` when we run `yarn build`?

Reply:
yep, removed